src/ex1/styles/ids.css

```diff
@@ -0,0 +1,59 @@
+#add-todo {
+  display: flex;
+}
+
+#new-todo-title:focus {
+  border: 2px solid var(--very-dark-green);
+  border-radius: 4px;
+  outline: none;
+}
+
+#new-todo-title {
+  padding: 5px 10px;
+  font-family: 'Indie Flower', cursive;
+  font-size: 20px;
+  width: 90%;
+}
+
+#all-todos-list {
+  font-family: 'Caveat', cursive;
+  padding-left: 0px;
+  border-radius: 4px;
+}
+
+#all-todos-list li {
```

Comment:
Using "li" here instead of a class makes it coupled to this specific solution. If you'll need to change li to a different element (ex: div) you'll have to make many changes in the code